### PR TITLE
rename `unifiedCacheState` to `unifiedState`

### PR DESCRIPTION
**Pull request by jennyf19**
fix spelling in xml

### CODE REVIEW

**Review by henrik-me — APPROVED** [review]
:shipit:

File: msal/src/Microsoft.Identity.Client/Features/TokenCache/TokenCacheExtensions.cs [hunk 1]
```diff
@@ -85,17 +85,17 @@ public static void SetBeforeWrite(this TokenCache tokencache, TokenCache.TokenCa
...
         /// </summary>
         /// <param name="tokenCache">Token cache to deserialize (to fill-in from the state)</param>
-        /// <param name="unifiedCacheState">Array of bytes containing serialized Msal cache data</param>
+        /// <param name="unifiedState">Array of bytes containing serialized Msal cache data</param>
         /// <remarks>
-        /// <paramref name="unifiedCacheState"/>Is a Json blob containing access tokens, refresh tokens, id tokens and accounts information
+        /// <paramref name="unifiedState"/>Is a Json blob containing access tokens, refresh tokens, id tokens and accounts information
         /// </remarks>
-        public static void Deserialize(this TokenCache tokenCache, byte[] unifiedCacheState)
+        public static void Deserialize(this TokenCache tokenCache, byte[] unifiedState)
```
Comment (bgavrilMS) [inline msal/src/Microsoft.Identity.Client/Features/TokenCache/TokenCacheExtensions.cs:92]: @jmprieur - This looks ok to me, do you have any comments on the naming of the params ?

Comment (jmprieur) [reply msal/src/Microsoft.Identity.Client/Features/TokenCache/TokenCacheExtensions.cs:92]: no, sorry. LGTM

**Review by jmprieur — APPROVED** [review]
LGTM

**Pull request merged**